diesel/src/macros/mod.rs

```diff
@@ -239,108 +263,235 @@ macro_rules! __diesel_column {
 /// ```
 #[macro_export]
 macro_rules! table {
-    // Put `use` statements at the end because macro_rules! cannot figure out
-    // if `use` is an ident or not (hint: It's not)
```

Comment:
This is still valid (the part about macro_rules and use).
But now this is not the main cause for moving import to the import field. Now this is done because it is easier to distinguish between import, doc and the table body in this way. So I think this comment is not needed anymore here.